### DISCUSSION TITLE
Facing error and unable to setup project in the local environment.

### DISCUSSION THREAD

**Discussion by m-shahzaib9** [taniarascia/takenote#562] [Q&A]
Hi,
I am unable to set up this project in my local environment, I have completed all the steps mentioned in the readme file. Please provide proper step-by-step documentation for setting up the development server. I will be very thankful to you. Please!!!

Check the screenshots, please.

![Screenshot from 2022-03-22 00-35-37](https://user-images.githubusercontent.com/62111307/159350816-33bac627-e94d-4cdf-868f-c86468343550.png)
![Screenshot from 2022-03-22 00-36-24](https://user-images.githubusercontent.com/62111307/159350836-b730dff7-db4a-4bb9-be80-46cee85416cf.png)
![Screenshot from 2022-03-22 00-36-46](https://user-images.githubusercontent.com/62111307/159350852-309cacc7-52c1-44da-808c-f226d645acd3.png)
![Screenshot from 2022-03-22 00-37-11](https://user-images.githubusercontent.com/62111307/159350876-a1608657-fe28-43f3-b0eb-a127fdd49538.png)

**Comment by Sukikiroi** [comment]
Hello Using the LTS version of Node (V16) rather than V17 fixed this issue for me

if that didn't work do this :
In your package.json: change this line
"start": "react-scripts --openssl-legacy-provider start"
